CRM/Upgrade/Incremental/sql/5.65.alpha1.mysql.tpl

```diff
@@ -1 +1,22 @@
 {* file to handle db changes in 5.65.alpha1 during upgrade *}
+
+-- Ensure new name field is not null/unique. Setting to ID is a bit lazy - but it works.
+UPDATE civicrm_group SET `name` = `id` WHERE name IS NULL;
+
+-- Add name field, make frontend_title required (in conjunction with php function)
+{if $multilingual}
+    {foreach from=$locales item=locale}
+      UPDATE `civicrm_group`
+      SET `frontend_title_{$locale}` = `title_{$locale}`,
+      `frontend_description_{$locale}` = `description`
+      WHERE `frontend_description_{$locale}` IS NULL OR `frontend_description_{$locale}` = '';
```

Comment:
@eileenmcnaughton my only thought is that it should be like below where we have 2 separate queries in case frontend_description is NOT NULL i.e. what if for some reason some one had set the frontend_description but not the frontend_title this SQL wouldn't run also why don't you have the same where clause as in L21 ?

Reply:
ah got it - I pushed the change up